elide-contrib/elide-dynamic-config-helpers/src/main/resources/templates/table.hbs

```diff
@@ -31,57 +33,57 @@ import javax.persistence.Id;
 /**
  * A root level entity for testing AggregationDataStore.
  */
-@Cardinality(size = CardinalitySize.{{#toUpperCase cardinality}}{{/toUpperCase}})
+@Cardinality(size = CardinalitySize.<%#toUpperCase cardinality%><%/toUpperCase%>)
 @EqualsAndHashCode
 @ToString
 @Data
-{{#if table}}@FromTable(name = "{{#if schema}}{{schema}}.{{/if}}{{table}}"){{/if}}
-{{#if sql}}@FromSubquery(sql = "{{sql}}"){{/if}}
-{{#if readAccess}}@ReadPermission(expression = "{{readAccess}}"){{/if}}
-{{#if description}}@Meta(description = "{{description}}"){{/if}}
-{{#if hidden}}@Exclude{{else}}@Include(rootLevel = true){{/if}}
-public class {{#capitalizeFirstLetter name}}{{/capitalizeFirstLetter}} {{#if extend}}extends {{#capitalizeFirstLetter extend}}{{/capitalizeFirstLetter}}{{/if}}{
+<%#if table%>@FromTable(name = "<%#if schema%><%schema%>.<%/if%><%table%>")<%/if%>
+<%#if sql%>@FromSubquery(sql = "<%sql%>")<%/if%>
+<%#if readAccess%>@ReadPermission(expression = "<%readAccess%>")<%/if%>
+<%#if description%>@Meta(description = "<%description%>")<%/if%>
+<%#if hidden%>@Exclude<%else%>@Include(rootLevel = true, type = "<%#capitalizeFirstLetter name%><%/capitalizeFirstLetter%>")<%/if%>
```

Comment:
for `type=`, instead of capitalizeFirstLetter let's use a new method lowerCaseFirstLetter so that its consistent with how we do when it's not dynamic config i.e. camelCase